### PR TITLE
feat: add message content intent faq

### PR DESCRIPTION
**Pull request by onerandomusername**
## Description

Adds guide information for Intents.message_content.

## Relevant Issues

To be merged with DisnakeDev/disnake#353

### CODE REVIEW

File: guide/docs/popular-topics/intents.mdx [hunk 1]
```diff
@@ -187,9 +192,29 @@ client = disnake.Client(intents=intents)
 # or
 from disnake.ext import commands
 
-bot = commands.Bot(command_prefix="!", intents=intents)
+bot = commands.Bot(command_prefix=commands.when_mentioned, intents=intents)
 ```
 
+### Why do most messages have no content?
+
+As of April 30th 2022, Discord has blocked message content from being sent to bots that do not declare the :attr:`Intents.message_content` intent when connecting to discord.
```
Comment (abhigyantrips) [inline guide/docs/popular-topics/intents.mdx:200]: Simply put, the Sphinx thingy wouldn't apply here.

```suggestion
As of April 30th 2022, Discord has blocked message content from being sent to bots that do not declare the <DocsLink reference="disnake.Intents.message_content">message_content</DocsLink> intent when connecting to discord.
```

File: guide/docs/popular-topics/intents.mdx [hunk 2]
```diff
@@ -187,9 +192,29 @@ client = disnake.Client(intents=intents)
...
+
+If you are on version 2.4 or before, your bot will be able to access message content without the intent enabled in the code. However, as of version 2.5, it is required to enable :attr:`Intents.message_content` to receive message content over the gateway.
```
Comment (abhigyantrips) [inline guide/docs/popular-topics/intents.mdx:202]: Again, Sphinx thingy.

```suggestion
If you are on version 2.4 or before, your bot will be able to access message content without the intent enabled in the code. However, as of version 2.5, it is required to enable <DocsLink reference="disnake.Intents.message_content">message_content</DocsLink> to receive message content over the gateway.
```

**Review by abhigyantrips — CHANGES_REQUESTED** [review]

File: guide/docs/popular-topics/intents.mdx (hunk 2 above)
Comment (onerandomusername) [reply guide/docs/popular-topics/intents.mdx]: https://github.com/DisnakeDev/guide/pull/30/commits/4911d5fe2ed73f6fc0c12ef596a55c7ff93fc2b8

File: guide/docs/popular-topics/intents.mdx (hunk 1 above)
Comment (onerandomusername) [reply guide/docs/popular-topics/intents.mdx]: https://github.com/DisnakeDev/guide/pull/30/commits/4911d5fe2ed73f6fc0c12ef596a55c7ff93fc2b8

**Comment by abhigyantrips** [comment]
This should ideally close #12 as well.

File: guide/docs/popular-topics/intents.mdx [hunk 3]
```diff
@@ -187,9 +192,35 @@ client = disnake.Client(intents=intents)
...
+
+As of April 30th 2022, Discord has blocked message content from being sent to bots that do not declare the <DocsLink reference="disnake.Intents.message_content">message_content</DocsLink> intent when connecting to discord.
+
+If you are on version 2.4 or before, your bot will be able to access message content without the intent enabled in the code. However, as of version 2.5, it is required to enable <DocsLink reference="disnake.Intents.message_content">message_content</DocsLink> to receive message content over the gateway.
+
+:::note
+
+No matter which disnake version you use, you will be required to turn on the message_content intent in the [discord development portal](https://discord.com/developers/applications).
+
+:::
```
Comment (abhigyantrips) [inline guide/docs/popular-topics/intents.mdx:208]:
```suggestion
:::note

No matter which disnake version you use, you will be required to turn on the message_content intent in the [Discord Developer Portal](https://discord.com/developers/applications).

:::
```

**Review by abhigyantrips — CHANGES_REQUESTED** [review]
Nitpick: Keep the capitalization same as the rest of the guide.

**Pull request merged**